src/engraving/rendering/dev/beamlayout.cpp

```diff
@@ -1260,6 +1272,29 @@ void BeamLayout::createBeamSegment(Beam* item, ChordRest* startCr, ChordRest* en
     }
 }
 
+int BeamLayout::computeStemShortenFactor(ChordRest* cr, Beam* beam, BeamSegment* beamSegment, int level,
```

Comment:
Isn't the term "factor" used usually for multipliers? Perhaps just "amount" is better here?